pallets/dapp-staking-migration/src/benchmarking.rs

```diff
@@ -21,7 +21,6 @@ use super::{Pallet as Migration, *};
 use astar_primitives::{dapp_staking::SmartContractHandle, Balance};
 use frame_benchmarking::{account as benchmark_account, v2::*};
 use pallet_dapp_staking_v3::StakeAmount;
-use sp_std::vec;
```

Comment:
I believe this had to be added in order for build to succeed. Otherwise compiler would complain that it's unclear which `vec` implementation is being used.

You don't see any issues with this?
E.g. `cargo check --features runtime-benchmarks`?

Reply:
That's what I was thinking as well. but `clippy` gives me an unused warning, and I don't see rust compiler actually complaining for any feature flag